modules/yieldmoBidAdapter.js

```diff
@@ -304,6 +304,16 @@ function openRtbImpression(bidRequest) {
   return imp;
 }
 
+function getBidFloor(bidRequest) {
+  let floorInfo = {};
+
+  if (typeof bidRequest.getFloor === 'function') {
+    floorInfo = bidRequest.getFloor({ currency: CURRENCY, mediaType: VIDEO, size: '*' });
```

Comment:
According to your bid parameters, you guys support bidfloor. It seems you are checking for video in the openrtb object for video but not support floors for banners. Is this intended?

Reply:
Thanks, this was a bug, fixed now.